src/test/java/org/junit/rules/RuleChainTest.java

```diff
@@ -93,4 +115,26 @@ public void whenRuleChainHasNullRuleTheStacktraceShouldPointToIt() {
         String stacktrace = Throwables.getStacktrace(result.getFailures().get(0).getException());
         assertThat(stacktrace, containsString("\tat org.junit.rules.RuleChainTest$RuleChainWithNullRules.<init>(RuleChainTest.java:"));
     }
+
+    public static class RuleChainWithMethodRules {
+        @Rule
+        public final RuleChain chain = RuleChain
+                .outerRule(new LoggingMethodRule("outer method rule"), RuleChainWithMethodRules.this)
```

Comment:
Please replace `RuleChainWithMethodRules.this` with simply `this` (here and in the example in the Javadoc)